torch/ao/quantization/qconfig.py

```diff
@@ -228,12 +228,19 @@ def get_default_qconfig(backend='x86', version=0):
     Returns the default PTQ qconfig for the specified backend.
 
     Args:
-      * `backend`: a string representing the target backend. Currently supports
+      * `backend` (str): a string representing the target backend. Currently supports
         `x86` (default), `fbgemm`, `qnnpack` and `onednn`.
 
     Return:
         qconfig
     """
+    supported_backends = ["fbgemm", "x86", "qnnpack", "onednn"]
+    if backend not in supported_backends:
+        raise AssertionError(
```

Comment:
Should this be `ValueError` instead?

Reply:
I actually checked this as well, ValueError means the type is correct and value is not correct, but in this case the type may not be correct either so I guess AssertionError is OK. we are also using AssertionError for the version number as well